astroquery/mast/observations.py

```diff
@@ -908,6 +908,45 @@ def service_request_async(self, service, params, *, pagesize=None, page=None, **
 
         return self._portal_api_connection.service_request_async(service, params, pagesize, page, **kwargs)
 
+    def mast_query(self, service, columns=None, **kwargs):
```

Comment:
we're trying to get rid of the ``**kwargs`` in all signatures, at least where it's not a convenience method falling back on totally different signatures (think about `Table.read()`), and where the list of potential kwargs are not unmanagably long. 

So, if possible, please spell them out.

Reply:
Also, please make optional agruments kwarg only. And please add documentation as docstring

Reply:
I think `**kwargs` are necessary here because `mast_query` deals with requests to all the different mashup services and uses kwargs for each service's list of properties. I also addressed the mashup property parameters using kwargs because they're all optional for the `Mast.service_request` call in the return statement. The api also does error-handling for properties or kwargs that don't exist within a service.

Reply:
Hi @bsipocz , Makayla is right. The arguments here depend on which database is being accessed, so we need `**kwargs` as a catchall. What do you mean by adding documentation as docstring? In this PR Makayla added a link to the MAST documentation a few lines below (L924 currently).